code/espurna/system.ino

```diff
@@ -75,6 +81,13 @@ unsigned long systemLoadAverage() {
     return _load_average;
 }
 
+void _systemSetupHeartbeat() {
+    if (getSetting("hbMode").length() == 0)  setSetting("hbMode", HEARTBEAT_MODE);
```

Comment:
Ok. That's I understood. But initially when first time device has been flashed we need to save those settings, otherwise always will be default value, not saved one. I'm wrong?